src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java

```diff
@@ -139,15 +141,104 @@ public static void deleteTempTargetBQTable() throws IOException, InterruptedExce
     }
   }
 
-  @After(order = 1, value = "@BQ_SINK_TEST_BQMT")
-  public static void deleteTempTargetBQMTTable() throws IOException, InterruptedException {
+  @After(order = 1, value = "@BQ_SINK_TEST_CSV_BQMT")
+  public static void deleteTempTargetBQMTCSVTable() throws IOException, InterruptedException {
     try {
-      BigQueryClient.dropBqQuery(PluginPropertyUtils.pluginProp("bqmtCsvTargetTable1"));
-      BigQueryClient.dropBqQuery(PluginPropertyUtils.pluginProp("bqmtCsvTargetTable2"));
-      BigQueryClient.dropBqQuery(PluginPropertyUtils.pluginProp("bqmtCsvTargetTable3"));
-      BeforeActions.scenario.write("BQ Target table - " + PluginPropertyUtils.pluginProp
-        ("bqmtCsvTargetTable1") + PluginPropertyUtils.pluginProp("bqmtCsvTargetTable2") +
-                                     PluginPropertyUtils.pluginProp("bqmtCsvTargetTable3") + " deleted successfully");
+      String[] targetTableToDelete = PluginPropertyUtils.pluginProp("bqmtCsvTargetTables").split(",");
+      for (String table : targetTableToDelete) {
+        BigQueryClient.dropBqQuery(table);
+        BeforeActions.scenario.write("BQ Target table - " + table + " deleted successfully");
+      }
+      bqTargetTable = StringUtils.EMPTY;
+    } catch (BigQueryException e) {
```

Comment:
add `try` `catch` block inside for loop.

Reply:
Added.